chart/open-feature-operator/README.md

```diff
@@ -126,24 +126,37 @@ The command removes all the Kubernetes components associated with the chart and
+| `controllerManager.kubeRbacProxy.resources.requests.cpu`                  | Sets cpu resource requests for kube-rbac-proxy.                                                                                  | `5m`                                         |
+| `controllerManager.kubeRbacProxy.resources.requests.memory`               | Sets memory resource requests for kube-rbac-proxy.                                                                               | `64Mi`                                       |
+| `controllerManager.manager.image.repository`                              | Sets the image for the operator.                                                                                                 | `ghcr.io/open-feature/open-feature-operator` |
+| `controllerManager.manager.image.tag`                                     | Sets the version tag for the operator.                                                                                           | `v0.5.4`                                     |
+| `controllerManager.manager.resources.limits.cpu`                          | Sets cpu resource limits for operator.                                                                                           | `500m`                                       |
+| `controllerManager.manager.resources.limits.memory`                       | Sets memory resource limits for operator.                                                                                        | `128Mi`                                      |
+| `controllerManager.manager.resources.requests.cpu`                        | Sets cpu resource requests for operator.                                                                                         | `10m`                                        |
+| `controllerManager.manager.resources.requests.memory`                     | Sets memory resource requests for operator.                                                                                      | `64Mi`                                       |
+| `controllerManager.replicas`                                              | Sets number of replicas of the OpenFeature operator pod.                                                                         | `1`                                          |
+| `managerConfig.flagsValidatonEnabled`                                     | Enables the validating webhook for FeatureFlag CR.                                                                               | `true`                                       |
+| `managerConfig.flagdResourceEnabled`                                      | Enables the controller for the Flagd CR and adds the required permissions to create/update/delete Service and Ingress resources. | `true`                                       |
```

Comment:
```suggestion
| `managerConfig.flagdResourceEnabled`                                      | Enables the controller for the Flagd CR and adds the required permissions to automatically manage the exposure of flagd via Service and Ingress resources. | `true`                                       |
```

Maybe you or somebody else can word this better, but I think a "why" was missing here, and I've tried to capture that.

Reply:
thanks for the suggestion, this makes things a bit clearer